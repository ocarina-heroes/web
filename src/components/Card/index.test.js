import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import Card from '.'

test('it matches snapshot', () => {
  const tree = renderer.create(<Card />).toJSON()
  expect(tree).toMatchSnapshot()
})
