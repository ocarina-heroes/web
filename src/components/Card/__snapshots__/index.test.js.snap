// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches snapshot 1`] = `
.c0 {
  margin: 1rem;
  -webkit-flex-basis: 45%;
  -ms-flex-preferred-size: 45%;
  flex-basis: 45%;
  padding: 1.5rem;
  text-align: left;
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
  border: 1px solid #eaeaea;
  border-radius: 10px;
  -webkit-transition: color 0.15s ease,border-color 0.15s ease;
  transition: color 0.15s ease,border-color 0.15s ease;
}

.c0 a:hover,
.c0 a:focus,
.c0 a:active {
  color: #0070f3;
  border-color: #0070f3;
}

.c0 h3 {
  margin: 0 0 1rem 0;
  font-size: 1.5rem;
}

.c0 p {
  margin: 0;
  font-size: 1.25rem;
  line-height: 1.5;
}

<a
  className="c0"
>
  <h3 />
  <p />
</a>
`;
